api/types/authentication.go

```diff
@@ -535,6 +536,72 @@ func (u *U2F) Check() error {
 			return trace.BadParameter("u2f configuration has an invalid attestation CA: %v", err)
 		}
 	}
+
+	// Removes the port from a URL host.
+	getDomain := func(s string) (string, error) {
+		if strings.Contains(s, ":") {
+			host, _, err := net.SplitHostPort(s)
```

Comment:
We do have our own version `utils.SplitHostPort()` if it makes sense to use that one instead.